### PR TITLE
Add sai call to isolate/unisolate a fabric port

### PR DESCRIPTION
**Pull request by jfeng-arista**
Add sai call for isolate/unisolate the fabric links.

Please merge this up after https://github.com/sonic-net/sonic-swss/pull/3132

**What I did**

**Why I did it**

**How I verified it**

**Details if related**

### CODE REVIEW

**Comment by jfeng-arista** [comment]
@kenneth-arista @arlakshm @vmittal-msft

File: orchagent/fabricportsorch.cpp [hunk 1]
```diff
@@ -785,6 +792,36 @@ void FabricPortsOrch::updateFabricDebugCounters()
...
+                    if (status != SAI_STATUS_SUCCESS)
+                    {
+                        SWSS_LOG_ERROR("Failed to set admin status");
+                    }
+                    SWSS_LOG_NOTICE("Set fabric port %d state done %d ", lane, isolated);
+                }
+            }
+            else
+            {
+                SWSS_LOG_NOTICE( "JFJF same isolation status for %d", lane);
```
Comment (arlakshm) [inline orchagent/fabricportsorch.cpp:823]: nit: fix this logs message?

Comment (jfeng-arista) [reply orchagent/fabricportsorch.cpp]: > nit: fix this logs message?

good catch. fixed. thank you !

**Review by arlakshm — APPROVED** [review]

**Comment by jfeng-arista** [comment]
tests failed with some "                'Invalid response from docker daemon: key "ApiVersion"' ' is missing.' . Not related to this chagne, will restart the tests pipeline

**Comment by jfeng-arista** [comment]
test_chassis_system_lag_id_allocator_table_full failed (1 runs remaining out of 2).
	<class 'AssertionError'>
	LAG ID allocator table full error is not returned
assert '0' == '1'

not related to this chagne

**Comment by jfeng-arista** [comment]
test_chassis_system_lag_id_allocator_table_full failed (1 runs remaining out of 2).
	<class 'AssertionError'>
	LAG ID allocator table full error is not returned
assert '0' == '1'

not  related to this chagne

**Comment by prsunny** [comment]
> test_chassis_system_lag_id_allocator_table_full failed (1 runs remaining out of 2). <class 'AssertionError'> LAG ID allocator table full error is not returned assert '0' == '1'
> 
> not related to this chagne

@jfeng-arista , looking into it

**Comment by jfeng-arista** [comment]
> > test_chassis_system_lag_id_allocator_table_full failed (1 runs remaining out of 2). <class 'AssertionError'> LAG ID allocator table full error is not returned assert '0' == '1'
> > not related to this chagne
> 
> @jfeng-arista , looking into it

thank you ! this failure seems affecting almost all PRs in swss .

**Comment by arlakshm** [comment]
@prsunny can you help merge this PR

**Comment by prsunny** [comment]
> @prsunny can you help merge this PR

@arlakshm , please check description. this is dependent on another PR. Or do you think this can go independently?

**Comment by arlakshm** [comment]
> > @prsunny can you help merge this PR
> 
> @arlakshm , please check description. this is dependent on another PR. Or do you think this can go independently?
Thanks @prsunny. I missed the dependency. @jfeng-arista, is this PR dependent on https://github.com/sonic-net/sonic-swss/pull/3132 ? or we can merge this first?

**Comment by jfeng-arista** [comment]
> > > @prsunny can you help merge this PR
> > 
> > 
> > @arlakshm , please check description. this is dependent on another PR. Or do you think this can go independently?
> > Thanks @prsunny. I missed the dependency. @jfeng-arista, is this PR dependent on #3132 ? or we can merge this first?

we can, just that it will not be protected under than toggle.

**Comment by jfeng-arista** [comment]
the failure not related to this change, it is synced down .

**Comment by jfeng-arista** [comment]
test failures are from upstream, not related to this change.

**Pull request merged**